setup.py

```diff
@@ -11,13 +11,21 @@
     if '@' not in r
 ]
 
+extra_requirements = {
+    "av": [
+        r
+        for r in Path("av_requirements.txt").read_text().splitlines()
+        if '@' not in r
+    ]
+}
+
 with open("README.md", encoding="utf8") as f:
     readme = f.read()
 
 
 setuptools.setup(
     name="augly",
-    version="0.1.3",
+    version="0.1.5",
```

Comment:
i think this should be version `0.1.4`, no?

Reply:
I already uploaded & then yanked a version 0.1.4 (the previous version of splitting dependencies into extras), so unfortunately that means I can't upload another 0.1.4.